changelog.md

```diff
@@ -1,3 +1,81 @@
+<a name="7.0.1"></a>
+# 7.0.1 (2021-01-26)
+[Full Changelog](https://github.com/GoogleChrome/lighthouse/compare/v7.0.0...v7.0.1)
+
+We expect this release to ship in the DevTools of [Chrome 90](https://chromiumdash.appspot.com/schedule), and to PageSpeed Insights within 2 weeks.
+
+## New contributors
+
+Thanks to our new contributors 👽🐷🐰🐯🐻!
+
+- Tyler Kindy @tkindy
+
+## Core
+
+* fr: add navigation runner ([#11975](https://github.com/GoogleChrome/lighthouse/pull/11975))
```

Comment:
All the "fraggle rock" stuff show be given less prominence, since it doesn't result in an actual feature the end user would care about today.

Maybe make a new header `Fraggle Rock`, move each there, plus give a link to the Fraggle Rock issue and a short line about what we're doing (auditing user flows) to hype it up? @patrickhulce  wdyt

Reply:
yeah separate header SGTM 👍  should I be tagging new commits with a `fr(scope):` moving forward and we create a new category in our generator?